fe/fe-core/src/main/java/com/starrocks/server/MetadataMgr.java

```diff
@@ -71,6 +73,9 @@ public MetadataMgr(LocalMetastore localMetastore, ConnectorMgr connectorMgr,
     }
 
     public Optional<ConnectorMetadata> getOptionalMetadata(String catalogName) {
+        if (Strings.isNullOrEmpty(catalogName)) {
+            catalogName = InternalCatalog.DEFAULT_INTERNAL_CATALOG_NAME;
+        }
```

Comment:
or  add check in isInternalCatalog ?

Reply:
The current change makes sense. It means if the input catalog-name is null, the internal catalog will be returned.
But this will change this function's behaviour, I suggest you add a comment for this functions behavior and make sure all the usages of this function won't be influenced by this change.

Reply:
done